src/Air.zig

```diff
@@ -583,6 +583,14 @@ pub const Inst = struct {
         /// Uses the `ty_pl` field.
         field_parent_ptr,
 
+        /// Implements @wasmMemorySize builtin.
+        /// Uses the `ty_pl` field, payload is `WasmMemoryIndex`.
```

Comment:
It should be ok to directly put the index in the `payload` field like `constant` does.

Reply:
I thought of that, but I felt like I was abusing the intent of the `ty_pl` & `pl_op` instructions. Considering we're already doing this for constants, I'll apply your suggestion instead. Thanks!